app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -134,12 +138,16 @@ public void onCreate(final Bundle savedInstanceState) {
         }
     }
 
+    public boolean isViewBinary(){
+        return !(getActivity() instanceof MainActivity);
```

Comment:
Yes this will move around and have a proper detection, it's just for development currently